src/staticdata.c

```diff
@@ -2284,6 +2308,7 @@ static void jl_save_system_image_to_stream(ios_t *f,
         jl_serialize_reachable(&s);
         // step 1.2: now that we have marked all bindings (badly), ensure all gvars are part of the sysimage
         record_gvars(&s, &gvars);
+        // TODO record external_fns?
```

Comment:
@timholy I think I need your help with this. I have a list of code_instances that I need to queue up for serialization. They should all be external.

Reply:
What do you want to do with them? The new external `CodeInstance`s are already "serialized" in https://github.com/JuliaLang/julia/blob/86e2130f14aa8204cbcef88cefa2e59d19248bd7/src/staticdata.c#L2278

Reply:
These are old code-instances that I need to be able to relocate against. Not doing anything seems to work so-far...

Reply:
Assuming they are `jl_object_in_image`, you should just be able to add `jl_queue_for_serialization(&s, old_instances_for_linking)` and then read it out with the same same shenanigans we use for `new_specializations`.